doc/api/child_process.md

```diff
@@ -1061,6 +1061,15 @@ pipes between the parent and child. The value is one of the following:
    corresponds to the index in the `stdio` array. The stream must have an
    underlying descriptor (file streams do not start until the `'open'` event has
    occurred).
+   **NOTE:** While it is technically possible to pass `stdin` as a writable or
+   `stdout`/`stderr` as readable under `options.stdio`, it is not recommended.
```

Comment:
nit: I don't think you need to need to explicitly state that this is under options.stdio, as this whole section is referring to that optional already